packages/app-degree-pages/src/components/ListingPage/components/ProgramList/ListView/index.colums.config.js

```diff
@@ -117,10 +117,10 @@ const columns = [
     sortable: true,
     contentTemplate: ({ resolver, actionUrls }) => (
       <div>
-        {resolver.getConcurrentDegrees().lenght > 0 && (
+        {resolver.getConcurrentDegrees().then(data => data).length > 0 && (
```

Comment:
Just double checking this function eventually calls fetchAcademicPlans which looks like it will alway return an array, do we need to handle if response is an error?